PsfLauncher/PsfPowershellScriptRunner.h

```diff
@@ -298,4 +288,38 @@ class PsfPowershellScriptRunner
 
 		return true;
 	}
+
+	LPPROC_THREAD_ATTRIBUTE_LIST MakeAttributeList()

```

Comment:
>MakeAttributeList [](start = 30, length = 17)

Do we want to do this at all anymore? I thought the issue here was that there is a behavioral difference between RS1 and RS2+.

Reply:
What are you asking, why we are making the attribute list for the scripts?

The issue that this PR is solving is to remove the option for scripts to be ran outside of the app container.

We are removing RS1 support because I could not get the scripts to not breakaway when using the RS1 SDK.

Reply:
So to be clear, we are dropping RS1 support in order to make all powershell scripts run inside the virtualized environment of the package (in container). We are also removing the ability for the powershell script to run outside of the helium container.

---
In reply to: [357374130](https://github.com/microsoft/MSIX-PackageSupportFramework/pull/106#discussion_r357374130) [](ancestors = 357374130)

Reply:
You are correct.